spec/operators/refCount-spec.ts

```diff
@@ -114,25 +114,4 @@ describe('refCount', () => {
     expect(arr[0]).to.equal('the number one');
     expect(arr[1]).to.equal('the number two');
   });
-
-  // TODO: fix firehose unsubscription
-  it.skip('should stop listening to a synchronous observable when unsubscribed', () => {
```

Comment:
Not sure why this one was just removed. Presumably because we're only fixing the modern operators?